### PR TITLE
Quote transform

### PR DESCRIPTION
**Pull request by aqk**
Please see
https://github.com/Chia-Network/clvm/pull/28
and
https://github.com/Chia-Network/clvm_tools/pull/16

### CODE REVIEW

**Comment by richardkiss** [comment]
There seem to be a lot of unrelated changes... does this need a rebase (or maybe rebase onto dev and do the PR to dev instead of main)?

File: src/full_node/mempool_check_conditions.py [hunk 1]
```diff
@@ -12,35 +12,7 @@
...
-GENERATOR_FOR_SINGLE_COIN_MOD = Program.from_bytes(
-    bytes.fromhex(
-        "ffff05ffff01ffffff05ff02ffff05ff02ffff05ffffff05ff05ffff01ff80808080f"
-        "fff05ff0bffff01ff8080808080808080ffff05ffff01ffffff05ffff04ff05ffff01"
-        "ffffff05ffff04ffff0aff11ff0b80ffff01ffff05ff49ffff05ff8200a9ffff01ff8"
-        "080808080ffff01ffffff05ff02ffff05ff02ffff05ff0dffff05ff0bffff01ff8080"
-        "80808080808080ff01808080ffff01ffff09808080ff01808080ff01808080"
-    )
-)
+from src.wallet.puzzles.generator_loader import GENERATOR_MOD, GENERATOR_FOR_SINGLE_COIN_MOD
```
Comment (richardkiss) [inline src/full_node/mempool_check_conditions.py:15]: You should be able to split this change out from the quote change.

**Comment by hoffmang9** [comment]
I took the liberty of re-targetting this to dev - which is where it should go.

File: src/full_node/mempool_check_conditions.py (hunk 1 above)
Comment (aqk) [reply src/full_node/mempool_check_conditions.py:15]: Thanks. Moved to here: https://github.com/Chia-Network/chia-blockchain/pull/729

**Pull request closed without merging**